File: SEM-5/PIP/Assignment-3/Question13.py
import math

def series_a(x,n):
    sum = 1
    for i in range(2,n+1,2):
        sum += (x**i)/math.factorial(i)
    return sum

def series_b(n):
    sum = 1
    for i in range(1,n+1):
        sum += (n**i)/math.factorial(i)
    return sum

ans = series_a(5,1)
print(ans)

File: SEM-5/OSW/Assignment-8/Question13.c
#include<stdio.h>

int sum(int [], int);

int main(){

    int a[5] = {1, 2, 3, 4, 5};
    int n = 5;
    printf("%d\n", sum(a, n-1));
    return 0;
}

int sum(int a[], int n){
    if(n==0) return a[0];
    return a[n] + sum(a, n-1);
}